File: HashTable/c++/Dictionary_HashTable/Word.cpp
#include <iostream>
#include "Word.h"
using namespace std;

Word::Word() {

}

Word::Word(string word, string type, string pronunciation, string example) {

	this->word = word;
	this->type = type;
	this->pronunciation = pronunciation;
	this->example = example;
}

string Word::getWord() {
	return word;
}

void Word::setWord(string word) {
	this->word = word;
}

string Word::getType() {
	return type;
}

void Word::setType(string type) {
	this->type = type;
}

string Word::getPronunciation() {
	return pronunciation;
}

void Word::setPronunciation(string pronunciation) {
	this->pronunciation = pronunciation;
}

string Word::getExample() {
	return example;
}

void Word::setExample(string example) {
	this->example = example;
}

string Word::toString() {
	return "Word [word=" + word + ", type=" + type + ", pronunciation=" + pronunciation + ", example=" + example
		+ "]";
}

File: HashTable/java/Dictionary_HashTable/src/com/cybersoft/hashtable/Word.java
package com.cybersoft.hashtable;

public class Word {
	private String word;
	private String type;
	private String pronunciation;
	private String example;

	public Word() {

	}

	public Word(String word, String type, String pronunciation, String example) {

		this.word = word;
		this.type = type;
		this.pronunciation = pronunciation;
		this.example = example;
	}

	public String getWord() {
		return word;
	}

	public void setWord(String word) {
		this.word = word;
	}

	public String getType() {
		return type;
	}

	public void setType(String type) {
		this.type = type;
	}

	public String getPronunciation() {
		return pronunciation;
	}

	public void setPronunciation(String pronunciation) {
		this.pronunciation = pronunciation;
	}

	public String getExample() {
		return example;
	}

	public void setExample(String example) {
		this.example = example;
	}

	@Override
	public String toString() {
		return "Word [word=" + word + ", type=" + type + ", pronunciation=" + pronunciation + ", example=" + example
				+ "]";
	}

}
